src/pages/Main/Main.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import styled from 'styled-components';
+import Nav from '../../components/Nav';
+import MainTopBanner from './MainTopBanner/Maintopbanner';
+
+const Main = () => {
+  return (
+    <div>
+      <Nav />
+      <MainTopBanner />
+    </div>
```

Comment:
```suggestion
    <>
      <Nav />
      <MainTopBanner />
    </>
```
스타일링이 필요한 부분이 아니라면 React.Fragment 사용해주시면 더 좋습니다!
목적에 맞게 수정해주세요~~ Main 부분이라 바로 수정을 안하더라도 혹시 몰라서 리마인드 드립니다!